### ISSUE TITLE
Reader assumes environment is little-endian

### ISSUE THREAD

**Issue by djlambert** [creof/wkb-parser#8]
`Reader::readDouble()` currently assumes it's running on a little-endian system when using the `d` format character to unpack a double. Logic is needed to support running on big-endian (SPARC, PPC, etc.) platforms.

**Comment by djlambert** [comment]
Confirmed things are not working on Solaris 11.3 SPARC (sun4v) with PHP 5.6.

```
PHPUnit 5.3.2 by Sebastian Bergmann and contributors.

...FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF  63 / 176 ( 35%)
FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF 126 / 176 ( 71%)
FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF.........FFFFF              176 / 176 (100%)

Time: 6.59 seconds, Memory: 20.00Mb

There were 164 failures:

(cut)

FAILURES!
Tests: 176, Assertions: 184, Failures: 164.
```

**Comment by djlambert** [comment]
#9 merged into develop for release 2.2
